src/node_util.cc

```diff
@@ -141,6 +141,12 @@ void Initialize(Local<Object> target,
   }
 #undef V
 
+  target->DefineOwnProperty(
+    env->context(),
+    OneByteString(env->isolate(), "pushValToArrayMax"),
+    Integer::NewFromUnsigned(env->isolate(), NODE_PUSH_VAL_TO_ARRAY_MAX),
+    v8::ReadOnly).FromJust();
```

Comment:
You could leave out the ReadOnly flag entirely since it's only a bindings object property.